src/core/Tooltip/style.ts

```diff
@@ -1,17 +1,19 @@
 import { css } from "@emotion/css";
 import {
-  AppThemeOptions,
   fontBodyXs,
   getColors,
   getShadows,
   getSpacings,
+  Props,
 } from "../styles";
 
-export const tooltipCss = (
-  { inverted = false },
-  theme: AppThemeOptions
-): string => {
-  const props = { theme };
+export interface ExtraProps extends Props {
+  inverted?: boolean;
+}
+
+export const tooltipCss = (props: ExtraProps): string => {
```

Comment:
I updated the css class helper functions to consolidate the props to accept just one argument `extraProps`, so the interface is identical to how we use `styled` to write CSS!